src/main/scala/com/typesafe/sbt/packager/debian/DebianPlugin.scala

```diff
@@ -66,25 +67,15 @@ object DebianPlugin extends AutoPlugin with DebianNativePackaging {
 
   val CHOWN_REPLACEMENT = "chown-paths"
 
-  def defaultMaintainerScript(name: String, replacements: Seq[(String, String)], tmpDir: File): Option[File] = {
-    val url = Option(getClass getResource s"$name-template")
-    url map { source =>
-      val scriptBits = TemplateWriter.generateScript(source, replacements)
-      val script = tmpDir / "tmp" / "etc" / "default" / name
-      IO.write(script, scriptBits)
-      script
-    }
-  }
-
   // TODO maybe we can put settings/debiansettings together
   /**
-   * Enables native packaging by default
-   */
-  override lazy val projectSettings = settings ++ inConfig(Debian)(debianSettings) ++ debianNativeSettings
+    * Enables native packaging by default
+    */
+  override lazy val projectSettings = settings ++ debianSettings ++ debianNativeSettings
 
   /**
-   * the default debian settings for the debian namespaced settings
-   */
+    * the default debian settings for the debian namespaced settings
+    */
   private def settings = Seq(

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/nepomukseiler/sbt-native-packager/file/1559303158/issues/source?bid=2040925&fileBranchId=2489531#l79)